std_msgs/Header header
mocap_ros/MocapDetection[] detections
